native_client/ctcdecode/scorer.cpp

```diff
@@ -12,22 +13,34 @@
 
 using namespace lm::ngram;
 
+static const int MAGIC = 'TRIE';
+static const int FILE_VERSION = 3;
+
 Scorer::Scorer(double alpha,
```

Comment:
For `Scorer::Scorer()` (and `Scorer::setup()` below) , maybe we should avoid changing upstream code and rely on overloading ? It looks like the signatures between our implem and theirs are different, and this would make it easier to follow and update in the future ?

Reply:
For this and the other comments about updating, I don't think it makes a big difference. This code is very likely to require more modifications in the future (already happening in #1675) so there's no expectation of just using this as a black box module.